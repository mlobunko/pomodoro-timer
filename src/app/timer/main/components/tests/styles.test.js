import React from "react";
import { shallow } from "enzyme";
import "enzyme-to-json";

import {
  StyledBackground,
  StyledCenterOfPage,
  StyledFront,
  StyledPage
} from "../styles";

describe("StyledBackground", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledBackground />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledCenterOfPage", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledCenterOfPage />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledFront", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledFront />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledPage", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
