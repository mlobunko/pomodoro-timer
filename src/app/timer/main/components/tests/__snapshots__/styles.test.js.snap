// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledBackground renders correctly 1`] = `
<div
  className="sc-bdVaJa eLhqDt"
/>
`;

exports[`StyledCenterOfPage renders correctly 1`] = `
<div
  className="sc-bwzfXH dObjBL"
/>
`;

exports[`StyledFront renders correctly 1`] = `
<div
  className="sc-htpNat csgGYq"
/>
`;

exports[`StyledPage renders correctly 1`] = `
<div
  className="sc-bxivhb cEVztx"
/>
`;
